src/client/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Avatar = (props) => {
+  return <div className="all-avatar-images">{props.image}</div>;
+};
+
+Avatar.defaultProps = {
+  image: PropTypes.image,
```

Comment:
there is no PropTypes image - read more here https://reactjs.org/docs/typechecking-with-proptypes.html
You should also define PropTypes first for example 

```
Avatar.propTypes = {
  src: PropTypes.string. isRequired
};
```